firmware/application/usb_serial_shell.cpp

```diff
@@ -1147,6 +1147,23 @@ static void cmd_asyncmsg(BaseSequentialStream* chp, int argc, char* argv[]) {
         chprintf(chp, usage);
     }
 }
+static void cmd_setfreq(BaseSequentialStream* chp, int argc, char* argv[]) {
```

Comment:
Would it be possible to also have a cmd_getfreq ?

Reply:
That is provided by the radioinfo iirc